src/System.ComponentModel.Composition/src/System/ComponentModel/Composition/CompositionException.cs

```diff
@@ -193,8 +193,10 @@ private static void WriteHeader(StringBuilder writer, int errorsCount, int pathC
             }
             else
             {
-                Assumes.IsTrue(errorsCount == 1);
-                Assumes.IsTrue(pathCount == 1);
+                if(errorsCount != 1 && pathCount != 1)
```

Comment:
this should be an OR instead of an AND operation.

Reply:
That's right! this was a slip.